CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Unreleased
+
+- feat: add support for optional `!` in scope (#12)
```

Comment:
This project uses Commitizen for automatically generating the CHANGELOG (`cz bump`). 

It's not a mandatory to update this manually, as the CHANGELOG will be compiled from your commit messages. (I should create a detailed `CONTRIBUTING.md` file to cover this process.)